test/filter_test.dart

```diff
@@ -71,25 +71,49 @@ void main() {
     }
   });
 
-  test('eq', () async {
-    final res = await postgrest
-        .from('users')
-        .select('username')
-        .eq('username', 'supabot');
-
-    for (final item in res as List) {
-      expect((item as Map)['username'] == ('supabot'), true);
-    }
-  });
-
-  test('neq', () async {
-    final res = await postgrest
-        .from('users')
-        .select('username')
-        .neq('username', 'supabot');
-    for (final item in res as List) {
-      expect((item as Map)['username'] == ('supabot'), false);
-    }
+  group("eq", () {
+    test('eq string', () async {
+      final res = await postgrest
+          .from('users')
+          .select('username')
+          .eq('username', 'supabot');
+
```

Comment:
Could we check that `res` is not empty for all tests that loops the res?

Reply:
Oh, great point!